glean-core/ios/Glean/Glean.swift

```diff
@@ -189,14 +188,6 @@ public class Glean {
 
     /// Handle background event and send appropriate pings
     func handleBackgroundEvent() {
```

Comment:
We removed that from the Android application, because we need to remove this API endpoint. Can you file a bug for removing this here too?

Reply:
API endpoint?  As in it was leaking into the public API?

Reply:
Also, do you have a link to the Android bug you are referring to so I can reference it in the new bug?

Reply:
Yes, sorry if I was unclear, my English-fu is weak today :) I meant "into the public API". The related bug is bug 1574470.

Reply:
Properties and functions are internal by default in iOS, so we shouldn't have to do anything.

Reply:
Ah! Right, I got confused again :) thanks for clarifying and sorry for the noise!